service/api.go

```diff
@@ -655,12 +656,16 @@ func (as *APIServer) validateProjectConfig(w http.ResponseWriter, r *http.Reques
 // LoggedError logs the given error and writes an HTTP response with its details formatted
 // as JSON if the request headers indicate that it's acceptable (or plaintext otherwise).
 func (as *APIServer) LoggedError(w http.ResponseWriter, r *http.Request, code int, err error) {
-	grip.Error(message.Fields{
+	errorFields := message.Fields{
 		"method": r.Method,
 		"url":    r.URL,
 		"err":    err,
```

Comment:
do "err.String()" here., I think this might end up printing some stack information in some situations